False positive on tests without subtests

The linter fails when a test doesn't have substests.

Code:

```go
package testdata

import (
	"testing"
)

func TestSomething(t *testing.T) {
	t.Parallel()
}
```

Error:

```
no_subtests_test.go:7:6: TestSomething's subtests should call t.Parallel
```

Expectation:

No errors, or at the very least a different error message.

--- a/testdata/src/test/without_sub_test.go
+++ b/testdata/src/test/without_sub_test.go
@@ -1,0 +1,10 @@
+package test
+
+import (
+	"testing"
+)
+
+func TestWithoutSub(t *testing.T) { // OK
+	t.Parallel()
+	call("TestWithoutSub")
+}

--- a/tparallel.go
+++ b/tparallel.go
@@ -40,6 +40,9 @@ func run(pass *analysis.Pass) (interface{}, error) {
 
 	testMap := getTestMap(ssaanalyzer, testTyp) // ex. {Test1: [TestSub1, TestSub2], Test2: [TestSub1, TestSub2, TestSub3], ...}
 	for top, subs := range testMap {
+		if len(subs) == 0 {
+			continue
+		}
 		isParallelTop := ssafunc.IsCalled(top, parallel)
 		isPararellSub := false
 		for _, sub := range subs {

